website/src/pages/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import styles from './styles.module.css';
+
+const features = [
+  {
+    title: 'Partition data into a hexagonal grid',
+    imageUrl: 'images/icon-high-precision.svg',
+    description: (
+      <>
+        H3 has an easy API for indexing coordinates into a hexagonal, global grid.
+      </>
+    ),
+  },
+  {
+    title: 'Hierarchical grid system',
+    imageUrl: 'images/icon-layers.svg',
+    description: (
+      <>
+        Easy, bitwise truncation to coarser, approximate cells, along with area compression/decompression algorithms.
+      </>
+    ),
+  },
+  {
+    title: 'Fully global',
+    imageUrl: 'images/icon-chart.svg',
+    description: (
+      <>
+        Along with twelve pentagons, the entire world is addressable in H3, down to square meter resolution.
```

Comment:
Nit: No need to bring the pentagons into this :)

"Aside from 12 ugly sores, H3 is blemish-free!"